src/main/java/com/example/MyBatisDemo/repository/StudentRepository.java

```diff
@@ -10,9 +10,9 @@
 import java.util.List;
 @Mapper
 public interface StudentRepository {
-    List<StudentEntity> findAllStudents();
+    List<StudentEntity> findAllStudents(@Param("name") String name,@Param("age") Integer age,@Param("city") String city);
```

Comment:
if name and paramName is same then do we need the @param annotation?

Reply:
It is not necessary to added
But stackoverflow blog says it is good practice to use @Pararm annotation.